python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -62,7 +79,117 @@ def __init__(self, config):
 
         cfg = RawIngestConfig()
         cfg.transfer = "direct"
-        self.task = RawIngestTask(config=cfg, butler=self.butler)
+        self.task = RawIngestTask(config=cfg,
+                                  butler=self.butler,
+                                  on_success=self.on_success,
+                                  on_ingest_failure=self.on_ingest_failure,
+                                  on_metadata_failure=self.on_metadata_failure)
+
+    def undef_metadata(self, filename):
+        """Return a sparsely initialized metadata dictionary
```

Comment:
Can't really find if it is a requirement, but in all examples in the LSST Dev Guide different sections of docstrings are separated by a blank line. So you may consider adding them here and in other docstrings below.

Reply:
Added